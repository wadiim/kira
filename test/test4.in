(0,0)+(0,1)=15
(0,2)+(1,2)=8
(1,0)+(1,1)+(2,0)+(3,0)=23
(2,1)+(2,2)+(2,3)=14
(0,3)+(1,3)=12
(0,4)+(1,4)=3
(2,4)+(2,5)+(2,6)=21
(0,5)+(0,6)+(0,7)=8
(1,5)+(1,6)+(1,7)=18
(0,8)+(1,8)=12
(2,7)+(2,8)+(3,7)=10
(4,0)+(4,1)+(3,1)+(3,2)+(5,1)+(5,2)=30
(4,2)+(4,3)+(3,3)+(5,3)+(4,4)+(4,5)=21
(3,4)+(3,5)+(3,6)=16
(5,4)+(5,5)+(5,6)=20
(4,6)+(4,7)=13
(3,8)+(4,8)+(5,8)=13
(5,0)+(6,0)+(7,0)+(7,1)=18
(8,0)+(8,1)=5
(6,1)+(6,2)+(6,3)=12
(7,2)+(8,2)=16
(7,3)+(8,3)=11
(6,4)+(6,5)+(6,6)=19
(7,4)+(8,4)=10
(7,5)+(7,6)+(7,7)=7
(8,5)+(8,6)+(8,7)=15
(5,7)+(6,7)+(6,8)=22
(7,8)+(8,8)=13
