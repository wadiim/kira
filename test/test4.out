786523149
592714863
341986752
678159234
915432678
423678591
164297385
859361427
237845916
